eta/modules/sample_videos.py

```diff
@@ -182,6 +197,27 @@ def _process_video(input_path, output_frames_dir, parameters):
         input_path, frames, output_patt=output_patt, size=size, fast=True)
 
 
+def _check_input_video_size(
+        stream_info, max_video_file_size, max_video_duration):
+    if max_video_file_size is not None:
+        file_size = int(stream_info.format_info["size"])
+        if file_size > max_video_file_size:
+            raise ValueError(
+                "Input video file size must be less than {}; found {}".format(
+                    etau.to_human_bytes_str(max_video_file_size),
+                    etau.to_human_bytes_str(file_size))
+            )
+
+    if max_video_duration is not None:
+        video_dur = float(stream_info.format_info["duration"])
```

Comment:
you should use `stream_info.duration` to get this. Use properties whenever possible